libs/external-router/driver/magento/src/transforms/resolution-to-resolvable-url.ts

```diff
@@ -5,6 +5,7 @@ import { MagentoUrlResolver } from '../model/url-resolver';
 export const transformResolutionToResolvableUrl = (
   resolution: MagentoUrlResolver,
 ): DaffExternallyResolvableUrl => ({
+  id: resolution.id?.toString() || resolution.entity_uid,
```

Comment:
I think this means we should now have two different transfomers

Reply:
@damienwebdev do you agree?

Reply:
Eh, I don't mind splitting the transformer one way or the other, the type is so similar as it's a single key. If this were to expand in later versions, I can foresee a split transformer, but I can't say with any confidence that splitting right now is valuable.

Reply:
well its done now anyway haha